Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:4.2.0
+
+ENV HOME=/home/app/
+
+COPY package.json $HOME/harvesterjs/
```

Comment:
as we are using shrinkwrap could we copy it as well right here? Because if something has changed on shrinkwrap the image should be recreated with these changes.